README.md

```diff
@@ -203,3 +204,7 @@ device_client.set_energy_forecast(<pv_energy_forecast_Wh>)
 ```
 
 ---
+## Interacting with the CLI
+You can run one of the setup scripts residing in the `d3a_api_client.tests` folder from the terminal.
+In order to run a module you can use the following command
+`d3a-api-client run --setup {module_name}`
```

Comment:
@Ravineesh Please be aware of this change of how to run the api-client (once it is pushed). Thanks!

Reply:
This is for running script locally, right? 

For Cypress (inside docker): 
![image](https://user-images.githubusercontent.com/6857904/105993086-3716a280-60a6-11eb-8f54-7613a49153b8.png)
where $imageName = d3a_api_client
Do I have to change this also ?

Reply:
@Ravineesh Yes, I will keep you updated when I push the changes so You can adapt and test accordingly

Reply:
```where $imageName = d3a_api_client```
No, as the images is build each time you start the tests, no need to change the image. Only the way of starting the api script.

Reply:
I have changed the predefined setups folder @Ravineesh

Reply:
ok, Thanks